api/client/secrets_v1_beta.go

```diff
@@ -101,15 +102,36 @@ func (c *SecretApiV1BetaApi) UpdateSecret(d *models.SecretV1Beta) (*models.Secre
 		identifier = d.Metadata.Name
 	}
 
+	fmt.Printf("This might overwrite the secret '%s' and change it's id.", identifier)
```

Comment:
I don't think this log belongs inside of this function. Also, we already have a more scary message in the command itself, so I think we can get rid of this